### PR TITLE
Popovers replaced by Tooltips in Category list in Cassiopeia #35467

### PR DESCRIPTION
**Pull request by charvimehradu**
Pull Request for Issue #35467.

### Summary of Changes
Changed popovers to tooltips

### Testing Instructions
Have a look on the category list in Cassiopeia with blog sample data

### Actual result BEFORE applying this Pull Request

131897635-fbc014c6-d871-4458-913e-3b7dfee86a75

### Expected result AFTER applying this Pull Request

test output

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Review by wilsonge — APPROVED** [review]

**Comment by brianteeman** [comment]
this is a library change that will impact everywhere that `HTMLHelper::_('grid.sort ` is used

@roland-d @obuisard I think that as a result this can only go in 4.3 but possibly only in 5

**Comment by roland-d** [comment]
This is definitely not for 4.2 as this is not a bugfix. The bigger question I guess is, do we want to change popovers to tooltips? In case we go to tooltips, we may want it everywhere for consistency.

@wilsonge Do you know/remember why we went to popovers? As I remember we used to have tooltips.

**Comment by chmst** [comment]
@roland-d I don't know why but in J4 we have only tooltips, no popovers. Maybe due to bootstrap jQuery.

There were only three popovers left in J4, one of them is this one. 
For me it is more a bugfix that a new feature.

**Comment by roland-d** [comment]
@chmst 
> There were only three popovers left in J4, one of them is this one.

Ok, good to know. In that case it is a bugfix indeed. We will need testers for this issue.

**Comment by ChristineWk** [comment]
screen shot 2023-02-18 at 15 13 11<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39643">issues.joomla.org/tracker/joomla-cms/39643</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on f0732246fa6cee06ada8dc114ba0cd2e158f0e95<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39643">issues.joomla.org/tracker/joomla-cms/39643</a>.</sub>

File: libraries/src/HTML/Helpers/Grid.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ abstract class Grid
     public static function sort($title, $order, $direction = 'asc', $selected = '', $task = null, $newDirection = 'asc', $tip = '', $form = null)
     {
         HTMLHelper::_('behavior.core');
-        HTMLHelper::_('bootstrap.popover', '.hasPopover', ['trigger' => 'hover focus']);
+        HTMLHelper::_('bootstrap.tooltip', '.hasTooltip', ['trigger' => 'hover focus']);
```
Comment (Quy) [inline libraries/src/HTML/Helpers/Grid.php]:
```suggestion
        HTMLHelper::_('bootstrap.tooltip', '.hasTooltip');
```
These are default triggers so not necessary to specify them.

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on c8459ee988ccfa4d2e30390499b773ba02b07b54<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39643">issues.joomla.org/tracker/joomla-cms/39643</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c8459ee988ccfa4d2e30390499b773ba02b07b54<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39643">issues.joomla.org/tracker/joomla-cms/39643</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39643">issues.joomla.org/tracker/joomla-cms/39643</a>.</sub>

**Comment by fancyFranci** [comment]
Thank you. It looks much better now :)

**Pull request merged**